paasta_tools/cli/cmds/spark_run.py

```diff
@@ -145,7 +145,19 @@ def add_subparser(subparsers):
     list_parser.add_argument(
         "-e",
         "--enable-compact-bin-packing",
-        help="Enabling compact bin packing will try to ensure executors are scheduled on the same nodes. Requires --cluster-manager to be kubernetes.",
+        help=(
+            "Enabling compact bin packing will try to ensure executors are scheduled on the same nodes. Requires --cluster-manager to be kubernetes."
+            " Always true by default, keep around for backward compability."
```

Comment:
are we planning on removing this flag once it’s no longer referenced in soaconfigs?

Reply:
Yes, at some point we're going to remove all `--enable-compact-bin-packing` from all of yelp-soa, but not today.